website/blog/2024-08-08-release-1.12.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Podman Desktop 1.12 Release
+description: Podman Desktop 1.12 has been released!
+slug: podman-desktop-release-1.12
+authors: cdrage
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.12/banner.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.12 Release! 🎉
+
+![podman-desktop-hero-1.12](img/podman-desktop-release-1.12/banner.png)
+
+Podman Desktop 1.12 is now available. [Click here to download it](/downloads)!
+
+This release includes:
+
+- **Podman remote**: We now support remote Podman setups! Manage your remote Podman machines all within the UI.
+- **macOS GPU support**: Container GPU access on macOS is now available. [`libkrun`](https://github.com/containers/libkrun) is now a selectable provider type to allow GPU passthrough enablement.
```

Comment:
We also do have the support for Windows.

Reply:
We do! But libkrun only applies for macOS which is added in this release. We need to update GPU in this issue: https://github.com/containers/podman-desktop/issues/8361

Reply:
Yeah, so I'd like to callout that we also added the support for GPU acceleration for Windows.